src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java

```diff
@@ -150,6 +151,15 @@ public String getItemID() {
         return id;
     }
 
+    /**
+     * Gets the {@link SlimefunItem} associated for this SlimefunItemStack. Null if no item is found.
+     *
+     * @return The {@link SlimefunItem} for this SlimefunItemStack, null if not found.
```

Comment:
Nitpicky but you could also mark "SlimefunItemStack" as a link, even though its the same file...

Reply:
Yeah... reee